src/System.Collections.Specialized/tests/HybridDictionary/ContainsObjTests.cs

```diff
@@ -3,425 +3,67 @@
-            }
-
-            //
-            for (int i = 0; i < len; i++)
-            {
-                // uppercase key
-                if (!hd.Contains(keysLong[i]))
-                {
-                    Assert.False(true, string.Format("Error, doesn't contain added uppercase \"{1}\"", i, keysLong[i]));
-                }
-
-                // lowercase key
-                if (hd.Contains(keysLong[i].ToUpper()))
-                {
-                    Assert.False(true, string.Format("Error, contains uppercase \"{1}\" - should not", i, keysLong[i].ToUpper()));
-                }
-            }
-
-            //  [] different_in_casing_only keys and Contains()
-            //
-
-            hd.Clear();
-            string[] ks = { "Key", "kEy", "keY" };
-            len = ks.Length;
-            for (int i = 0; i < len; i++)
-            {
-                hd.Add(ks[i], "Value" + i);
-            }
-            if (hd.Count != len)
-            {
-                Assert.False(true, string.Format("Error, count is {0} instead of {1}", hd.Count, len));
-            }
-
-            if (hd.Contains("Value0"))
-            {
-                Assert.False(true, string.Format("Error, returned true when should not"));
-            }
-            for (int i = 0; i < len; i++)
-            {
-                if (!hd.Contains(ks[i]))
-                {
-                    Assert.False(true, string.Format("Error, returned false when true expected", i));
-                }
-            }
-
-            cnt = hd.Count;
-            len = valuesLong.Length;
-            for (int i = 0; i < len; i++)
-            {
-                hd.Add(keysLong[i], valuesLong[i]);
-            }
-            if (hd.Count != len + cnt)
-            {
-                Assert.False(true, string.Format("Error, count is {0} instead of {1}", hd.Count, len + cnt));
-            }
-            // verify new keys
-            for (int i = 0; i < len; i++)
-            {
-                if (!hd.Contains(keysLong[i]))
-                {
-                    Assert.False(true, string.Format("Error, doesn't contain \"{1}\"", i, keysLong[i]));
-                }
-            }
-            // verify old keys
-            for (int i = 0; i < ks.Length; i++)
-            {
-                if (!hd.Contains(ks[i]))
-                {
-                    Assert.False(true, string.Format("Error, doesn't contain \"{1}\"", i, ks[i]));
-                }
-            }
-
-
-            //
-            //   [] Contains(null) - for filled dictionary
-            //
-            len = valuesShort.Length;
-            hd.Clear();
-            for (int i = 0; i < len; i++)
-            {
-                hd.Add(keysShort[i], valuesShort[i]);
-            }
-
-            Assert.Throws<ArgumentNullException>(() => { hd.Contains(null); });
-
-            // [] Contains() for case-insensitive comparer dictionary
```

Comment:
I didn't see any case-sensitivity checks in the new code. Did I miss them again?